docs/source/modules.rst

```diff
@@ -1,14 +1,5 @@
-.. initially generated by
-   sphinx-apidoc -f -o source ../ ../versioneer.py ../setup.py
-   (everything after the first "../" are exclude patterns)
-
-PyFstat module documentation
-============================
-
-These pages document the full API and set of classes provided by PyFstat.
-
-See :doc:`here <readme>` for installation instructions and
-other general information.
+PyFstat
+=======
```

Comment:
still/again messed up

Reply:
Done in 14eaf8a6905c67b3f9797298ea2cc691a57243be